CMakeLists.txt

```diff
@@ -1,10 +1,23 @@
 # Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
 # Licensed under the MIT license
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 
 project(OsciView VERSION 1.0.0)
 
+if(WIN32)
+    if(NOT DEFINED SFML_HOME)
+        if(DEFINED ENV{SFML_HOME})
+            set(SFML_HOME $ENV{SFML_HOME})
+        else()
+            message(FATAL_ERROR "SFML_HOME is not defined. Please, set it with -DSFML_HOME=path/to/sfml in cmake command, or add an environment variable in Windows, or set it from your IDE settings.")
+        endif()
+    endif()
+
+    message("Using SFML_HOME=${SFML_HOME}")
```

Comment:
Just an idea: This call to `message` could use [`STATUS`](https://cmake.org/cmake/help/latest/command/message.html) for a message class.

Reply:
Done.
If you don't see anything else, I can merge the PR. 🙂

Reply:
@BenPyton let's go! :smiley: